homeassistant/components/hive/__init__.py

```diff
@@ -112,13 +104,11 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     return unload_ok
 
 
-def refresh_system(
-    func: Callable[Concatenate[_HiveEntityT, _P], Awaitable[Any]]
-) -> Callable[Concatenate[_HiveEntityT, _P], Coroutine[Any, Any, None]]:
+def refresh_system(func):
```

Comment:
Please make sure you rebase properly on latest dev branch. I don't think we should remove the typing here.